### PR TITLE
Refine Conference Program [WIP]

### PR DESCRIPTION
**Pull request by dcwalk**
A couple things seem like they should happen:

- [x] Some visual differentiation on session type -- attempted, not sure if it looks great
- [x] Add in session details (and speaker info)
- [x] Link to fuller session descriptions
- [x] Links to speakers within page
- [x] Links to social media
  - [x] handle odd lines (instagram in data? multiple links?)
- [x] add `rel="noopener"` to links
- [x] update presentations/speakers
  - [x] vmon
  - [x] jenny

Stretch:
- Generate an ICS (MY FAVE 📆 )

### CODE REVIEW

**Comment by dcwalk** [comment]
screen shot 2018-06-03 at 10 02 26 am

Just a quick ☝️ check in on how the session/speaker details are starting to look

**Comment by dcwalk** [comment]
Yooooo - @garrying  I think I've been doing bad practice with sending a href to new windows (shoot-shoot), can you weigh in a preferred practice?

**Comment by garrying** [comment]
@dcwalk I think the practice makes sense in scenarios where we anticipate a visitor will not want to lose their place on a page, and is accessing peripheral information such as social media pages. Looks like we've been consistent in new-page-opening of social media links in 2017 and location links, which I think we can stick with! One thing to add is `rel=noopener` attribute https://mathiasbynens.github.io/rel-noopener/

**Comment by benhylau** [comment]
Current branch doesn't have descriptions and we should hide social media icons that are not provided:

screen shot 2018-06-04 at 2 58 36 pm

Can we also link from box in the schedule to its description?

Is the plan to manually create `_data/sessions.yml`?

**Comment by dcwalk** [comment]
It's WIP, I have stuff locally to commit.
The info is all in the body of the issue so yeah, will be manual and link into gh issues for more detail

**Comment by dcwalk** [comment]
Hey so pushed my local work @benhylau and @garrying improvements welcome!
I think dis-aggregating sessions and presenters probably a good idea...?

**Comment by benhylau** [comment]
A few issues remain:
- [x] I have no idea why the `:` in `Keynote: To Be Announced` get dropped in the `##` link so the linking is broken, we use the same line to process the title:
  `{{ session.title | truncatewords: 6, '' | downcase | replace: ' ', '-' }}`
- [x] When clicking from schedule to jump to a session, the top bar covers top of content
- [x] We need to add more spacing between sections in the details, and style the text for better readability

**Comment by dcwalk** [comment]
Hey @benhylau thanks for taking a pass! I'm finding the session break out a little confusing (before it was alphabetized so that seemed a little easier to browse), I'm wondering if we could revert to having one list but then make clear the different session types thru visual cues or a "Format: XXX" 

Just thinking about how people will be interacting with this: I expect they'll start with the program and click into names, it doesn't seem like knowing all the workshops because they are beside each other offers some additional findability unless I'm missing something

**Comment by benhylau** [comment]
@dcwalk I found the previous single-list grouping confusing because:

* Two items with similar names look like duplicates, it's only clear to organizers that they are workshops following talks
* These types of workshops have no descriptions, throwing off readability of the list (this format allows us to paste same description twice)
* `Lighting Talks` was an item on its own, which is just a title by itself making me think the things that follow are lightning talks (the current one allows linking from table to the Lightning Talk header)
* `Keynote` was also awkwardly mixed into the alphabetical sorting, I think it should instead be emphasized at the top

**Comment by dcwalk** [comment]
It looks like at this point all the session types are groups together except for the edge cases, why not just handle those three you identify seperately via filters?

- listing the lightning talks under the heading and suppressing from showing up in the for loop (this is already partially implemented)
- filtering out Keynote (also can remove :to be announced from title and move to description)
- could have all split session links go to a single description or keep as two in the way you have

File: conference/program.md [hunk 1]
```diff
@@ -230,3 +230,43 @@ redirect_from:
 </tr>
 </tbody>
 </table>
+
+## Keynote: To Be Announced
+
+## Talks & Discussions
```
Comment (dcwalk) [inline conference/program.md]: Should update this title to something more general to the 4 session types

**Comment by benhylau** [comment]
@dcwalk so how is what you suggest different from what we have now? You just want workshops to be mixed in with talks?

I find it nice to browse all workshops in a group, since those are the ones I need to pick which track. So it does help with that navigation model. Also if we mix the two categories we still have the first two issues I raised.

**Comment by dcwalk** [comment]
The workshops are not the only parallel sessions, a mix of other ones run in parallel:
- Artists body workshop 
- Protocol Tactics
- Money and Mesh

**Comment by benhylau** [comment]
Thanks @garrying you make everything look good again. I added back Sprint Pitches and made a fix to header anchors.

@dcwalk I merged Workshops back into Sessions.

If this looks good we can go ahead and merge.

**Comment by benhylau** [comment]
Oh wait, still need to link speakers... and what is this?
>Links to social media
>handle odd lines (instagram in data? multiple links?)

**Comment by garrying** [comment]
There's a session with an Instagram link, which I think we can accommodate. Dealing with multiple links beyond the set of 3 we have (link, GitHub, Twitter) seems like something we could address later.

I'm not too sure how to go about linking the multi-speaker sessions. Maybe revising the way `presenter` is constructed?

**Comment by benhylau** [comment]
@garrying Yea I turned it into a list. Have a look 9bde92fe4bea1042dad65bb40a3547e837c0c574

**Comment by garrying** [comment]
@dcwalk @benhylau Made a few more edits to clean up targeting. Are we good to merge?

**Review by benhylau — APPROVED** [review]

**Pull request merged**